pkg/ingester/ingester.go

```diff
@@ -689,6 +732,10 @@ func (i *Ingester) AllUserStats(ctx context.Context, req *client.UserStatsReques
 
 // Check implements the grpc healthcheck
 func (i *Ingester) Check(ctx context.Context, req *grpc_health_v1.HealthCheckRequest) (*grpc_health_v1.HealthCheckResponse, error) {
+	if err := i.checkRunningOrStopping(); err != nil {
```

Comment:
The readiness probe below (`CheckReady()`) only consider `Running` valid, while here we consider `Stopping` too. Which one is right? Shouldn't they be aligned?

Reply:
I don't see any issue for `CheckReady` using `checkRunningOrStopping` check as well.